kolibri/core/content/utils/import_export_content.py

```diff
@@ -185,7 +185,7 @@ def get_import_export_nodes(  # noqa: C901
             nodes_query = nodes_query.filter(renderable_contentnodes_q_filter)
 
         # filter down the query to remove files associated with nodes we've specifically been asked to exclude
-        if exclude_node_ids:
+        if exclude_node_ids is not None:
```

Comment:
Never mind, this made sense in the end. I did need to change some tests, but this ended up a useful distinction for the case where a manifest file needs to indicate that there is no content. Just needed a bit more documentation :)

So what's happening here is, if you run `importcontent` without specifying any node_ids, this function gets called with `node_ids=None`. That does the right thing and matches with all nodes in the channel. There was some funny behaviour before, where if you do `importcontent --node_ids=''`, this function ends up being called with `node_ids=['']`, which also does the right thing (it matches matches with no nodes in the channel), but in the wrong way. But that's great, because it means cleaning this up so `None` and `[]` are different doesn't break the UI!

So in the end I didn't revert this, but I added an explanatory comment in `get_import_export_nodes`, and I added some unit tests for the specific cases where `node_ids=[]` and `node_ids=None`.